slices/ca-certificates.yaml

```diff
@@ -3,5 +3,10 @@ package: ca-certificates
 slices:
   data:
     contents:
-      /etc/ssl/certs/:
-      /usr/share/ca-certificates/mozilla/*.crt:
+      /etc/ssl/certs/ca-certificates.crt: {text: FIXME, mutable: true}
+      /usr/share/ca-certificates/mozilla/: {until: mutate}
+      /usr/share/ca-certificates/mozilla/*: {until: mutate}
+    mutate: |
+      certs_dir = "/usr/share/ca-certificates/mozilla/"
+      certs = [ content.read(certs_dir + path) for path in content.list(certs_dir) ]
```

Comment:
Nitpick: I think it's more conventional in Python to not have spaces padding the edges of list comprehensions. Is that true? I don't mind too much the details of our conventions, as long as it's consistent, but since we're just starting we should try to borrow some typical Python style.

Reply:
Ack. Fixed.